styles/prosilver/template/event/viewtopic_body_post_buttons_list_before.html

```diff
@@ -0,0 +1 @@
+<!-- DEFINE $SHOW_POST_BUTTONS = (postrow.POST_NUMBER != 1) -->
```

Comment:
I guess it should be ```DEFINE $SHOW_POST_BUTTONS = ($SHOW_POST_BUTTONS and postrow.POST_NUMBER != 1)```, otherwise the post button will be shown with no respect of ```postrow.U_EDIT or postrow.U_DELETE or postrow.U_REPORT or postrow.U_WARN or postrow.U_INFO or postrow.U_QUOTE``` condition.